src/ppx/knob.cpp

```diff
@@ -13,11 +13,21 @@
 // limitations under the License.
 
 #include "ppx/knob.h"
+#include "ppx/string_util.h"
 
 #include <cstring>
 
 namespace ppx {
 
+// Spacing:
+//
+// --flag_name <params>    description...
+//                         continued description...
+// |usageMsgIndentWidth---|
+// |usageMsgTotalWidth-----------------------------|
+constexpr size_t usageMsgIndentWidth = 20;
+constexpr size_t usageMsgTotalWidth  = 80;
```

Comment:
Could you add "k" prefix to the const? 
kUsageMsgIndentWidth and kUsageMsgTotalWidth